pyrk/materials/flibe.py

```diff
@@ -6,6 +6,8 @@
 class Flibe(Material):
     """This class represents FLiBe. It inherits from the material
     class and possesses attributes intrinsic to flibe.
+    All properties from the report: Temperature-Dependent Thermophysical Properties
+    for Fluoride Salts and Simulant Fluids[TH lab at UCB]
```

Comment:
Thanks for adding this. However, this report doesn't appear to be public. It isn't here: http://fhr.nuc.berkeley.edu/references/ucb-th-report-series/ . Do you have a URL? I generally prefer either archival journal articles or publicly available documents where possible.